### DISCUSSION TITLE
7.1 does not shutdown frozen even for days at hp logo screen

### DISCUSSION THREAD

**Discussion by abbazs** [elementary/triage#583] [Q&A]
I've installed elementary os 7.1 in HP zbook. And it doesnot shutdown or reboot, it just stays like this in the image even for days. Is there a solution for this?
![image](https://github.com/user-attachments/assets/3afea811-ed76-468b-adf0-ebed033e8ec6)

**Comment by sudanchapagain** [comment]
I never waited for days but when i had not closed few applications and tried to shutdown it had gotten stuck for a long while too. I just pressed the power button for 15 seconds which force shutdowns and after that i never experienced it.
